WordPress/Classes/ViewRelated/Me/App Settings/MediaCacheSettingsViewController.swift

```diff
@@ -0,0 +1,124 @@
+import Foundation
+
+class MediaCacheSettingsViewController: UITableViewController {
+    fileprivate var handler: ImmuTableViewHandler!
```

Comment:
I know this descends from how `ImmutableViewHandler` is structured, but maybe we can use a regular optional instead of implicitly unwrap, and have the assignment `handler = ImmuTableViewHandler(takeOver: self)` moved over in the initializer, right after the `super.init(_:)` call?